soc/arm/nxp_imx/rt/soc.c

```diff
@@ -35,12 +35,10 @@ const clock_usb_pll_config_t usb1PllConfig = {
 #endif
 
 #ifdef CONFIG_ETH_MCUX_0
+/* ENET PLL configuration for RUN mode */
 const clock_enet_pll_config_t ethPllConfig = {
-	.enableClkOutput0 = true,
-	.enableClkOutput1 = false,
-	.enableClkOutput2 = false,
-	.loopDivider0 = 1,
-	.loopDivider1 = 1
+	.enableClkOutput500M = true,
+	.loopDivider = 1,
```

Comment:
From what I get from the code and the reference manual, it uses the the ENET PLL (PLL6) configured to 500 MHz to run the ARM core. I guess it would be better to use the SYS PLL (PLL2) with a PFD set to 18 instead. I guess that can be done with `CLOCK_InitSysPfd()`.

Reply:
could you please explain why it's better to use sys_pll? both ENET and sys pll can be used as the source of the core.

Reply:
Shouldn't we keep ENET for the Ethernet device? Also the above code will likely break support for the IMXRT1050/60. The code should work for all SoC.

Reply:
ENET PLL is not dedicate for ENET from SoC aspect. Could you please let me know how it will break the support for IMXRT1050/60?

Reply:
it is conditioned in config file.

Reply:
> Oh i just saw that you pushed a new version. Thanks, now it doesn't remove any code. The last remaining question is to know if enabling the 500 MHz clock is fine for the other SoCs. They also enable CONFIG_ETH_MCUX_0.



I've updated the macro to CONFIG_INIT_ENET_PLL in both Kconfig.soc and soc.c, since it is what "CONFIG_ETH_MCUX_0" was doing in soc.c at least.